src/components/Filter/FilterActiveLabel.js

```diff
@@ -0,0 +1,17 @@
+import cx from 'classnames';
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const FilterActiveLabel = ({ className, title, ...rest }) => {
+  const classes = cx('filter-pf-active-label', className);
+  return <p className={classes}>{title}</p>;
+};
+
+FilterActiveLabel.propTypes = {
+  /** Additional css classes */
+  className: PropTypes.string,
+  /** Title for the list (ie. 'Active Filters:') */
+  title: PropTypes.string.isRequired
```

Comment:
I know that {title} is typically just a string... but {children} is more flexible. What if I want to put an emoji in there and a <b> tag...

Reply:
another option is to just use PropTypes.node and leave it named {title}...

Reply:
title  is bad anyhow, causes unwanted tooltips at times, I will update